build.gradle

```diff
@@ -8,7 +8,6 @@ import org.jetbrains.gradle.ext.Gradle
 plugins {
     id 'java'
     id 'java-library'
-    id 'maven-publish'
```

Comment:
Also just a leftover?

Reply:
We aren't using maven publishing right now, so it just reduces the dependencies to download for the time being. In the future we can easily re-add it.